mesop/components/bokeh_plot/bokeh_plot.py

```diff
@@ -0,0 +1,29 @@
+from io import BytesIO
+from typing import Protocol
+from bokeh.plotting import figure
+from mesop.component_helpers import Style, component
+from mesop.components.html.html import html
+from bokeh.embed import file_html
+
+
+class figure(Protocol):
```

Comment:
figure -> Figure

Reply:
The class name in bokeh library starts with small case [figure class github](https://github.com/bokeh/bokeh/blob/ce4ab47a86197e3742d82cbe7cb4565f840a7210/src/bokeh/plotting/_figure.py#L94)
do you still recommends the change?

Reply:
ah, I missed that. in that case let's keep it as-is, thanks.